app/controllers/orders_controller.rb

```diff
@@ -1,11 +1,20 @@
 class OrdersController < ApplicationController
+
+
+
   def index
-    @user = current_user
+    @user = current_user #ok because we call @user.orders to show the orders. BUT users can still get to other pages.
```

Comment:
@BLaurenB what do you mean by this comment?

Reply:
Oh, sorry, I forgot to take it out as I was playing around with how to limit users' access to orders!  You can totally delete it, or you can send it back to me to update.